BankSDK/GiniBankSDKExample/Tests/IntegrationTests/TransferSummary/SkontoTransferSummaryIntegrationTest.swift

```diff
@@ -0,0 +1,222 @@
+
+        // Validate skonto discounts if applicable
+        validateSkontoDiscounts(fixtureContainer: fixtureExtractionsAfterFeedbackContainer,
+                                extractionResult: extractionResult)
+
+        // Cleanup
+        GiniBankConfiguration.shared.cleanup()
+        XCTAssertNil(GiniBankConfiguration.shared.documentService)
+        
+        expect.fulfill()
+    }
+
+    func validateSkontoDiscounts(fixtureContainer: ExtractionsContainer,
+                                 extractionResult: ExtractionResult) {
+        let fixtureSkontoDiscounts = fixtureContainer.compoundExtractions?.skontoDiscounts?.first
+        if let fixtureSkontoDiscountsAfterFeedback = extractionResult.skontoDiscounts?.first {
+            XCTAssertEqual(fixtureSkontoDiscounts?.first(where: { $0.name == "skontoAmountToPayCalculated" })?.value,
+                           fixtureSkontoDiscountsAfterFeedback.first(where: { $0.name == "skontoAmountToPayCalculated" })?.value)
+        }
+    }
+
+    func createAmountExtraction(value: String) -> Extraction {
+        Extraction(box: nil,
+                   candidates: nil,
+                   entity: "amount",
+                   value: value,
+                   name: "amountToPay")
+    }
+}
+
+// MARK: - Specific Handlers
+
+class SkontoTransferSummaryHandler: BaseSkontoTransferSummaryHandler<SkontoTransferSummaryIntegrationTest> {
+
+    override func applySkontoChangesIfNeeded(result: AnalysisResult) {
+        guard let skontoDiscountExtraction = result.skontoDiscounts?.first else { return }
+
+        let updatedAmountToPayString = "1000.00:EUR"
+        let updatedPercentage = "50.0"
+
+        // Modify result extractions
+        result.extractions["amountToPay"]?.value = updatedAmountToPayString
+        let modifiedSkontoExtractions = skontoDiscountExtraction.map { extraction -> Extraction in
+            let modifiedExtraction = extraction
+            switch modifiedExtraction.name {
+            case "skontoAmountToPay", "skontoAmountToPayCalculated":
+                modifiedExtraction.value = updatedAmountToPayString
+            case "skontoPercentageDiscounted", "skontoPercentageDiscountedCalculated":
+                modifiedExtraction.value = updatedPercentage
+            default:
+                break
+            }
+            return modifiedExtraction
+        }
+        
+        GiniBankConfiguration.shared.skontoDiscounts = [modifiedSkontoExtractions]
+    }
+
+    override func sendTransferSummary(result: AnalysisResult) {
+        guard let updatedAmountToPayString = result.extractions["amountToPay"]?.value else { return }
+        let amountExtraction = createAmountExtraction(value: updatedAmountToPayString)
+        GiniBankConfiguration.shared.sendTransferSummaryWithSkonto(amountToPayExtraction: amountExtraction,
+                                                                   amountToPayString: updatedAmountToPayString)
+    }
```

Comment:
[clarification] Why do we do this here:
`  guard let updatedAmountToPayString = result.extractions["amountToPay"]?.value else { return }`  and in the same method but for `SkontoNotAppliedTransferSummaryHandler` we don't do this?
If there is no difference can we have just one?

Reply:
@ValentinaIancu-Gini thats true, I've checked that after some changes (during development) I have same overrides for both specific handlers and they are the same as default implementation in base handler. I removed this and left only base implementation for both, thank you for raising it!